server/routes/testutils/appSetup.ts

```diff
@@ -21,12 +21,13 @@ jest.mock('../../services/auditService')
 export const user: HmppsUser = {
   name: 'FIRST LAST',
   userId: 'id',
-  token: 'token',
+  token:
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkZpcnN0IExhc3QiLCJpYXQiOjE1MTYyMzkwMjIsImF1dGhvcml0aWVzIjpbIlJPTEVfQUxFUlRTX1JFRkVSRU5DRV9EQVRBX01BTkFHRVIiLCJST0xFX01BTkFHRV9TRUNVUkVfQUxFUlRTIl19.GhXs8pD74dIh8nl6O9RWbAKrDok-wWqPydhQl9qnUxw',
```

Comment:
Did you mean to put this in here? Do you need to rotate a token now? :D

Reply:
It looks like a faked JWT with the right roles but yes I also wondered why it was in there

Reply:
yes i put a real jwt here for the unit tests to pass, because i'm not mocking the middleware to check for user role.
i think i'll be able to take this away after i revamp the jest tests into e2e tests, so i don't want to put too much effort int this and just hardcode a jwt for now.